Partition (WBP)/Functions/VI0035A.VTI0035.fun.yml

```diff
@@ -514,6 +514,7 @@
     Clr_List Named(#wl_VTTEMP_Function)
     
     #Com_Owner.mthAddFunction( VI0035C )
+    #Com_Owner.mthAddFunction( VI0035D )
```

Comment:
This needs to be
```
If (#LF_VTTEMP_Skip_Test_Case = False)
#Com_Owner.mthAddFunction( VI0035D )
Else
#LF_VTTEMP_Number_Of_Skipped_Test_Case += 1
Endif
```

Reply:
Okay , Will add it as above

Reply:
This has been incorporated with below commit.